x-pack/plugins/infra/public/apps/logs_app.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ApolloClient } from 'apollo-client';
+import { History } from 'history';
+import { CoreStart } from 'kibana/public';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Route, Router, Switch } from 'react-router-dom';
+import { AppMountParameters } from '../../../../../src/core/public';
+import '../index.scss';
+import { NotFoundPage } from '../pages/404';
+import { LinkToLogsPage } from '../pages/link_to/link_to_logs';
+import { LogsPage } from '../pages/logs';
+import { ClientPluginDeps } from '../types';
+import { createApolloClient } from '../utils/apollo_client';
+import { CommonInfraProviders, CoreProviders } from './common_providers';
+import { prepareMountElement } from './common_styles';
+
+export const renderApp = (
+  core: CoreStart,
+  plugins: ClientPluginDeps,
+  { element, history }: AppMountParameters
+) => {
+  const apolloClient = createApolloClient(core.http.fetch);
```

Comment:
[off-topic] Do we still need an `apolloClient` in logs? Do we have any graphQL lingering around?

Reply:
yeah, I think the link-to routes use the old source. They are tricky to remove, because the `host|container|pod-logs` route needs access to the metrics ui host/container/pod field config, which is not part of the new HTTP+JSON source config API anymore. I'm planning to change those links from the metrics ui to use the generic `filter` query param instead. Good eye, though :wink: